app-admin/logstash-bin/logstash-bin-7.16.2.ebuild

```diff
@@ -32,9 +32,9 @@ src_prepare() {
 	default
 
 	local d
-	for d in aarch64-Linux arm-Linux Darwin i386-Linux i386-SunOS \
+	for d in aarch64-FreeBSD aarch64-Linux arm-Linux Darwin i386-Linux i386-SunOS \
 		i386-Windows mips64el-Linux ppc64-AIX ppc64le-Linux ppc64-Linux \
-		ppc-AIX sparcv9-Linux sparcv9-SunOS x86_64-DragonFlyBSD \
+		ppc-AIX s390x-Linux sparcv9-Linux sparcv9-SunOS x86_64-DragonFlyBSD \
```

Comment:
nice

Reply:
What do you mean by that?

Reply:
Just well spotted that upstream gained support for it!

Reply:
Actually it's pretty harmless, just installs a file that won't be used. But it was in the portage output log and Ago opened a bug for it, so why not fixing it when it's Xmas time :)